guides/common/modules/snip_red-hat-helper-apps.adoc

```diff
@@ -0,0 +1,11 @@
+ifdef::upgrading-connected,upgrading-disconnected[]
+For interactive upgrade instructions, you can use the {ProjectName} Upgrade Helper on the Red{nbsp}Hat Customer Portal.
+This application provides you with an exact guide to match your current version number.
```

Comment:
"Exact guide to match your version number" seems like a very odd way to say this. Wouldn't it be better to use something like "...provides upgrade instructions relevant for your current version of {ProjectName}"?

Reply:
It would be better! Applied (but I'm going with "customized" rather than "relevant").